src/iac_scan_runner/scan_runner.py

```diff
@@ -66,6 +78,22 @@ def init_checks(self):
         snyk = SnykCheck()
         sonar_scanner = SonarScannerCheck()
 
+        # This matrix should be revised and extended, it is just a proof of concept here as for now
```

Comment:
TODOs should be marked as the IDE will mark them too.

```suggestion
        # TODO: This matrix should be revised and extended, it is just a proof of concept here as for now
```

Reply:
Ok, this one is really to be extended, so it lies under TODO category